conda/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "$PREFIX" ]; then
+  PREFIX="$CONDA_PREFIX"
+fi
+
+# conda build will copy everything over, including build directories.
+# Don't let this pollute the build!
+rm -rf build || true
```

Comment:
I still don't really understand what this is for.

Reply:
A cmake build system will conventionally have its build products placed in the `build` subdirectory. If this directory is not removed, cmake will make use of cached settings; the best way to get a clean build is to make sure there is no `build` to begin with.

So now, the problem is that `conda-build` starts a build by doing a full recursive copy of EVERY file in your source tree, including things you'd consider build products. That means that if you had a leftover `build` directory, conda will accidentally grab it. That's bad. So we delete it.

BTW, if you've ever had conda-build take a really long time to start up, that might be because you have some big binaries in your source directory. Delete 'em and try again.

Reply:
So this is just to workaround running `anaconda-build` in your directory if you already have an existing cmake build setup?

Reply:
Yep.